pajbot/oauth_client_edit.py

```diff
@@ -0,0 +1,133 @@
+import logging
+
+from json import loads as jsonify
+import base64
+import requests
+
+from flask import request, session
+from flask_oauthlib.client import OAuth, parse_response
+from flask_oauthlib.client import OAuthRemoteApp
+from flask_oauthlib.client import OAuthResponse
+from flask_oauthlib.client import OAuthException
+
+from oauthlib.common import add_params_to_uri
+from flask_oauthlib.utils import to_bytes
+
+
+log = logging.getLogger(__name__)
+
+
+class OAuthEdited(OAuth):
```

Comment:
This solution does not seem nice, especially not without proper reasoning behind it